public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Airbnb Item Page</title>
+  </head>
+  <body>
+    <div id="Details"></div>
+    <script src="http://localhost:3003/1/bundle.js"></script>
```

Comment:
Is this script src tag always supposed to retrieve the bundle for propertyID 1? I was thinking that the bundle would differ a little bit for each property ID, or maybe I'm wrong...Otherwise, looks great!

Reply:
this is something i should clean up. the propertyId in the script src tag is not really used for anything/it doesnt retrieve a specific bundle. the server for my details service returns static assets from '/:propertyId' as shown below. i don't know how to fix at this point. need to figure it out

`
app.use('/:propertyId', express.static(path.join(__dirname, '../public')));
`